fecgov/fec-cms
# Make menu_title optional, restore it show in Wagtail page explorer

## Summary (required)

- Resolves #4811
- Make menu_title optional, restore it show in Wagtail page explorer
- This reverts a recent change to  menu_title display that was done to make it easier for editors to keep track of copied pages in the Wagtiail page explorer.
PR:  #4785  
Issue: #https://github.com/fecgov/fec-cms/issues/4416 
However this is reverted here because the menu_title was never showing up.

In order for  Wagtail to  show a new  menu_title for a copied page in the Wagtail page explorer, the page has to have been published at least once. So a draft page that has never been published will not show the new menu_title. Although there are ways to modify the `copy-page` behavior  in `wagtail_hooks.py`, there is no reasonable way to solve the copy issue considering the low priority level agreed upon by the team for this request.  Editors will just have to continue to publish and then unpublish copied pages to see the correct menu_title.



### Required reviewers
one frontend, one content

## Impacted areas of the application

	new file:   home/migrations/0121_auto_20211206_1255.py
	modified:   home/models.py


## How to test

- Checkout branch and `./manage.py migrate`
-  Create a ResourcePage. CustomPage , or CollectionPage
       - Add a menu_title and see that it shows up in Wagtail explorer upon saving-draft  or publishing
       - Leave menu_title blank and see that it reverts to showing the page title instead
- Copy a ResourcePage. CustomPage , or CollectionPage, then open the new page for editing
       - Confirm that you can show the menu_title in Wagtail explorer after editing and publishing
       - Confirm that menu_title persists in page explorer if you unpublish the page back to draft.
